roles/science/tasks/mrtrix.yml

```diff
@@ -37,6 +16,35 @@
       - libtiff5-dev
       - libpng-dev
```

Comment:
lets add git here since its needed for the clone

We also need the complete list of dependencies here:
https://mrtrix.readthedocs.io/en/3.0.4/installation/build_from_source.html#linux

This kind of dev needs to be tested on a empty machine (VM) so that we make sure its not "your system" that happens to already have everything it needs.

Reply:
Ok. I think we were missing git, g++ and python-is-python3 form the list.  python-is-python3 will be installed with afni anyways but I'll add it here too.

Reply:
I'm gonna test the changes in a fresh VM only using mrtrix3 tag.